### DISCUSSION TITLE
Add performance execution time and memory usage after submit.

### DISCUSSION THREAD

**Discussion by sshgun** [codewars/codewars.com#3006] [Ideas]
Hi folks, nice application, a great change that can improve your site is adding other details after the submit of the answers, like the execution time, memory usage. and set ranks for these aspects.

As code analyst y use the site to practices basic stuff, and review the others users solutions, so will be nice to get that detail of memory, execution time at the end.

**Comment by SoftzillaAT** [comment]
I also want to measure my solution against others in terms of exection time. This would be a great challenge to get the solution with the lowest exection time.
